Project-3/Jvonderh Project 3

```diff
@@ -0,0 +1,220 @@
+  (e)  Inverse Functional
+  Has characteristic
+  This should be inverse functional because, in a relationship where a bearer has a characteristic which depends on the bearer for its existence, there will be more than one characteristic which the bearer will have. Since an inverse functional object property requires that there be only one input in the form of a characteristic into the bearer, this will mean that the bearer can only bear one characteristic. As an example, it is possible for a person to bear the characteristics of "is 6 feet tall" and "has a beard." This will mean that the has characteristic relation will have more than a value of 1 or 0. As such, the has characteristic relation should not be inverse functional. 
+```
+
+3. Model the following natural language expressions using terms from BFO and RO; you are welcome to introduce new terms where needed:  
+```
+  (a) Sally has an arm Tuesday but does not have an arm Wednesday.
+  Sally is instance_of object
+Tuesday is instance_of one-dimensional temporal region 
+Wednesday is an instance of one-dimensional temporal region 
+Arm is an instance of fiat object part
+“Sally participates in having at least one arm on Tuesday” is an instance of occurrent 
+“Sally participates in having no arms on Wednesday” is an instance of occurrent
+“Sally participates in having at least one arm on Tuesday precedes Sally participates in having no arms on Wednesday”
+
+  (b) Every liver has some cell as part at all times it exists.
+  Liver has_part_at_all_times Cell
+  
+  (c) John was a child, then an adult, then a senior. 
+  John is an instance_of object
+childhood is an instance_of occurrent.
+adulthood is an instance_of occurrent.
+seniorhood is an instance_of occurrent.
+“John participates in childhood precedes John participates in adulthood which precedes John participates in seniorhood.”
+
+  (d) Goofus and Gallant are married at each point in a three year span. 
+  Goofus is an instance_of object
+Gallant is an instance_of object
+Marriage is an instance_of occurent
+“Three years span 1” is an instance_of one-dimensional temporal region.
+If zero-dimensional temporal region t1 is part_of the one-dimensional temporal region “three years span 1”, then Goofus participates in marriage at t1 and Galland participates in marriage at t1.
+
+```
+
+4. Using the language of First-Order Logic, represent the following natural language expressions; you are welcome to introduce new terms where needed: 
+```
+  (a) Sally has an arm Tuesday but does not have an arm Wednesday.
+  ∃(x)(Sx^(Tx^~Wx)) 
+  S=Is Sally 
```

Comment:
Generally speaking, a proper name like "Sally" is treated as a constant rather than a predicate in logical translation. So a more natural treatment is to say "x=s" (some x is just Sally) instead of "Sx" (x instantiates Sally-hood).

Here you treat Tuesday and Wednesday as individuals. But given there are many Tuesdays or Wednesday, it might be better to treat them as predicates like Tx (x is a Tuesday) or Wx (x is a Wednesday).

Reply:
That makes sense overall. I changed the first part of the conjunction to be that equivalence relation and adjusted the "S" to fit with that.

I will keep the current formulation with the days, as there is nothing in the sentence that requires us to represent that there are multiple Tuesdays and Wednesdays. We can represent that as you suggested, but I think it is fine to not do so.